packages/swapper/src/utils.ts

```diff
@@ -191,3 +194,19 @@ export const checkEvmSwapStatus = async ({
     return createDefaultStatusResponse(txHash)
   }
 }
+
+export const getRate = ({
+  sellAmountCryptoBaseUnit,
+  buyAmountCryptoBaseUnit,
+  sellAsset,
+  buyAsset,
+}: {
+  sellAmountCryptoBaseUnit: string
+  buyAmountCryptoBaseUnit: string
+  sellAsset: Asset
+  buyAsset: Asset
+}): BigNumber => {
+  const sellAmountCryptoHuman = fromBaseUnit(sellAmountCryptoBaseUnit, sellAsset.precision)
+  const buyAmountCryptoHuman = fromBaseUnit(buyAmountCryptoBaseUnit, buyAsset.precision)
+  return bn(buyAmountCryptoHuman).div(sellAmountCryptoHuman)
```

Comment:
nitpick - return `toFixed()` and remove `toFixed()` and `toString()` from portals and one inch getTradeQuote respectively

Reply:
fd57bc186c0fc99f05f446572882093b0a5b68a6